parts/k8s/kubernetesinstalls.sh

```diff
@@ -197,9 +200,9 @@ installImg() {
 extractHyperkube() {
     CLI_TOOL=$1
     path="/home/hyperkube-downloads/${KUBERNETES_VERSION}"
-    mkdir -p "$path"
     pullContainerImage $CLI_TOOL ${HYPERKUBE_URL}
     if [[ "$CLI_TOOL" == "docker" ]]; then
+        mkdir -p "$path"
```

Comment:
Pre-creating the directory works for the docker extraction implementation (`docker run ...` below), but it was breaking the non-docker extraction implementation (`img unpack...` below). This change fixes #184.